### PR TITLE
Shopping feature

### PR DESCRIPTION
**Pull request by MadayAlcala**

### CODE REVIEW

File: src/test/java/steps/LoginSteps.java [hunk 1]
```diff
@@ -36,7 +36,7 @@
     @Given("The user goes to \"([^\"]*)\" page")
     public void initializeLoginPage(final String url) {
         PageTransporter.goToUrl(url);
-        loginPage = new LoginPage();
+
```
Comment (titor-oopart) [inline src/test/java/steps/LoginSteps.java]: delete empty line

File: src/main/java/AutomationPractice/ui/Common/UrlBuilder.java [hunk 2]
```diff
@@ -29,6 +29,9 @@
     static final String SUMMER_DRESSES = "id_category=11&controller=category";
     static final String EVENING_DRESSES = "id_category=10&controller=category";
     static final String CONTROLLER_ORDER = "controller=order";
+    static final String T_SHIRTS = "id_category=5&controller=category";
+    static final String BLOUSES = "id_category=7&controller=category";
+
```
Comment (titor-oopart) [inline src/main/java/AutomationPractice/ui/Common/UrlBuilder.java:34]: delete empty line

File: src/test/java/steps/LoginSteps.java (hunk 1 above)
Comment (MadayAlcala) [reply src/test/java/steps/LoginSteps.java]: Solved.

File: src/main/java/AutomationPractice/ui/Common/UrlBuilder.java (hunk 2 above)
Comment (MadayAlcala) [reply src/main/java/AutomationPractice/ui/Common/UrlBuilder.java:34]: Solved.

**Review by titor-oopart — APPROVED** [review]

**Pull request merged**